misc/scripts/remove.sh

```diff
@@ -85,6 +85,8 @@ case "$url" in
 	;;
 esac
 
+sudo dpkg -r "$name-pacstall" # removes virtual .deb package
```

Comment:
This should be inside the `*)` case after, removing package, with a fancy message about removing the dummy